tracer/slog.go

```diff
@@ -196,3 +188,40 @@ func replacePlaceholders(query string, args []interface{}) string {
 
 	return buffer.String()
 }
+
+func formatArg(arg interface{}) string {
+	var res string
+
+	val := reflect.ValueOf(arg)
+	if val.Kind() == reflect.Ptr {
+		if val.IsNil() {
+			res = "<nil>"
+		} else {
+			res = formatArg(val.Elem().Interface())
+		}
+
+		return res
+	}
+
+	switch v := arg.(type) {
+	case bool:
+		res = fmt.Sprintf("%t", v)
+	case int, uint, int8, uint8, int16, uint16, int32, uint32, int64, uint64:
+		res = fmt.Sprintf("%d", v)
+	case float64, float32:
+		res = fmt.Sprintf("%f", v)
+	case []uint8:
+		if len(v) == 0 {
+			res = "0x"
+		} else {
+			hexString := hex.EncodeToString(v)
+			res = fmt.Sprintf("'0x%s'", hexString)
```

Comment:
It might make sense to use `"'\x%s'"`, since Postgres can handle this syntax `'\x5425890298aed601595a70ab815c96711a31bc65'` as a binary input format